package.json

```diff
@@ -43,7 +43,7 @@
     "launchdarkly-js-client-sdk": "^2.18.1",
     "lodash": "^4.17.20",
     "regenerator-runtime": "^0.13.7",
-    "sbc-common-components": "2.4.46",
+    "sbc-common-components": "2.4.48",
```

Comment:
Why not update all the way to 2.4.52?

You'll have to look at what's new between '46 and '52 and possibly verify anything that might break or function differently.

Reply:
Sure. I'll use 2.4.52 and fix things that break. Thanks.

Reply:
Hopefully, nothing will break.

If something does break, it may be out of scope for this ticket. Up to you to decide, or we can discuss it and figure it out.

Reply:
I checked the release notes and they are minor tweaks for the navigation and log out. I've also run checks to see if any functionality breaks. It was fine so far, for the create and filing UIs. I'll get this PR ready for review once I checked out everything.